scopesim/tests/test_scratch.py

```diff
@@ -0,0 +1,15 @@
+import scopesim as sim
+from astropy import units as u
+
+import os
+
+def test_lms():
+    PKGS_DIR = os.path.abspath("../../../irdb/")
```

Comment:
This will fail on github, should we xfail it? Or move it to the IRDB tests?

Reply:
That seems to be your work (commit 4362186 2022-03-02). I think the file name calls for the file to be scratched...

Reply:
Ok, hmm. Not sure why there's an LMS test in there though. Leave it for now, and when I go through and greenify the tests, I'll either delete it or move it.